dev/io.openliberty.microprofile.faulttolerance.3.0.internal_fat_tck/publish/tckRunner/tck/tck-suite-lite.xml

```diff
@@ -115,23 +115,23 @@
             <class name="org.eclipse.microprofile.fault.tolerance.tck.config.FallbackConfigTest"></class>
             <class name="org.eclipse.microprofile.fault.tolerance.tck.config.FallbackSkipOnConfigTest"></class>
             <class name="org.eclipse.microprofile.fault.tolerance.tck.interceptor.FaultToleranceInterceptorTest"></class>
-            <class name="org.eclipse.microprofile.fault.tolerance.tck.metrics.AllMetricsTest"></class>
-            <class name="org.eclipse.microprofile.fault.tolerance.tck.metrics.BulkheadMetricTest">
-                <methods>
-                    <include name="bulkheadMetricTest"/>
-                    <include name="bulkheadMetricRejectionTest"/>
-                </methods>
-            </class>
-            <class name="org.eclipse.microprofile.fault.tolerance.tck.metrics.CircuitBreakerMetricTest"></class>
-            <class name="org.eclipse.microprofile.fault.tolerance.tck.metrics.ClassLevelMetricTest"></class>
-            <class name="org.eclipse.microprofile.fault.tolerance.tck.metrics.FallbackMetricTest"></class>
-            <class name="org.eclipse.microprofile.fault.tolerance.tck.metrics.MetricsDisabledTest"></class>
-            <class name="org.eclipse.microprofile.fault.tolerance.tck.metrics.RetryMetricTest"></class>
-            <class name="org.eclipse.microprofile.fault.tolerance.tck.metrics.TimeoutMetricTest">
-                <methods>
-                    <include name="testTimeoutMetric"/>
-                </methods>
-            </class>
+<!--             <class name="org.eclipse.microprofile.fault.tolerance.tck.metrics.AllMetricsTest"></class> -->
+<!--             <class name="org.eclipse.microprofile.fault.tolerance.tck.metrics.BulkheadMetricTest"> -->
+<!--                 <methods> -->
+<!--                     <include name="bulkheadMetricTest"/> -->
+<!--                     <include name="bulkheadMetricRejectionTest"/> -->
+<!--                 </methods> -->
+<!--             </class> -->
+<!--             <class name="org.eclipse.microprofile.fault.tolerance.tck.metrics.CircuitBreakerMetricTest"></class> -->
+<!--             <class name="org.eclipse.microprofile.fault.tolerance.tck.metrics.ClassLevelMetricTest"></class> -->
+<!--             <class name="org.eclipse.microprofile.fault.tolerance.tck.metrics.FallbackMetricTest"></class> -->
+<!--             <class name="org.eclipse.microprofile.fault.tolerance.tck.metrics.MetricsDisabledTest"></class> -->
+<!--             <class name="org.eclipse.microprofile.fault.tolerance.tck.metrics.RetryMetricTest"></class> -->
+<!--             <class name="org.eclipse.microprofile.fault.tolerance.tck.metrics.TimeoutMetricTest"> -->
+<!--                 <methods> -->
+<!--                     <include name="testTimeoutMetric"/> -->
+<!--                 </methods> -->
+<!--             </class> -->
```

Comment:
I know it's explained in the PR, but do you think it's worth having a comment here about why these tests are commented out?